notebooks/classification_2d_mednist.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<p>isn't a dataset like this availabke within monai?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I don't think so (at least for now). This dataset is from their MedNIST notebook but I couldn't find it in their Repo</p>